### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.text.ui)

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2331]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/beta#editor)
#### Element path
button[title="Refresh\ extension\ microturtle"] > .text.ui
#### Snippet
`<span class="ui text">v0.0.9</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.84 (foreground color: #ffffff, background color: #999999, font size: 10.3pt, font weight: bold). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

this is the version number for github extensions that have been added:

<img width="232" alt="Screen Shot 2019-08-02 at 2 39 57 PM" src="https://user-images.githubusercontent.com/5615930/62400262-773da900-b533-11e9-9be5-54f02068cf65.png">
